RELEASE-NOTES.txt

```diff
@@ -6,6 +6,7 @@
 -----
 * [internal] Fixed an issue where source and platform tags were not added to a Zendesk ticket if the account has no blogs. [#17084]
 * [*] Set the post formats to have 'Standard' first and then alphabetized the remaining items. [#17074]
+* [*] Fixed wording of theme customization screen's menu bar by using "Activate" on inactive themes. [#17060]
```

Comment:
@illusaen note that this entry won't make it into the release notes in the App Store, because those have already been submitted for translation by #17131.

Given the content, I think that's okay, but let me know if you think otherwise.